src/components/AnnotationsPane.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Utility } from '../lib/Utility';
+
+class AnnotationsPane extends React.Component {
+  renderAnnotationInProgress() {
+    if (!this.props.annotationInProgress) return null;
+
+    const svgPointPrefix = 'ANNOTATION_IN_PROGRESS_POINT_';
+    const svgPoints = [];
+
+    for (let i = 0; i < this.props.annotationInProgress.points.length; i += 1) {
+      const point = this.props.annotationInProgress.points[i];
+      svgPoints.push(
+        <circle
+          key={svgPointPrefix + i}
+          cx={point.x}
+          cy={point.y}
+          r={10}
+          fill="#20FECB"
```

Comment:
Reminder: use constants instead of hardcoded style values. OK for a future cleanup PR.

Reply:
Are you referring to the hex code here? I can't get the variable from `app-theme.styl`, but I can put a constant in this component.

Reply:
👍 Exactly what I was thinking. Maybe `./src/styles.js`